layouts/partials/sign-up-node-operator-email/form.html

```diff
@@ -47,6 +47,7 @@ <h2 class="light-black">What’s your email?</h2>
             email = input.value
         }
         analytics.identify(email, {email: email, authToken: "unknown"});
+        analytics.identify(analytics.user().anonymousId(), {email: email, sno_register_email_page_visited: true});
```

Comment:
this call is pretty the same as a previous one, but i have to ad a bit different data scheme so i have added a new call, not to break previous flow